ncm-ceph/src/main/pan/components/ceph/schema.pan

```diff
@@ -208,23 +208,26 @@ type ceph_mds = {
 
 @documentation{ ceph cluster-wide config parameters }
 type ceph_cluster_config = {
-    'fsid'                      : string # Should be generated with uuidgen
-    'filestore_xattr_use_omap'  : boolean = true
-    'osd_journal_size'          : long(0..) = 10240
-    'mon_initial_members'       : string [1..]
-    'public_network'            : type_network_name
-    'cluster_network'           ? type_network_name
-    'auth_service_required'     : string = 'cephx' with match(SELF, '^(cephx|none)$')
     'auth_client_required'      : string = 'cephx' with match(SELF, '^(cephx|none)$')
     'auth_cluster_required'     : string = 'cephx' with match(SELF, '^(cephx|none)$')
-    'osd_pool_default_pg_num'   : long(0..) = 600
-    'osd_pool_default_pgp_num'  : long(0..) = 600
-    'osd_pool_default_size'     : long(0..) = 3
-    'osd_pool_default_min_size' : long(0..) = 2
+    'auth_service_required'     : string = 'cephx' with match(SELF, '^(cephx|none)$')
+    'cluster_network'           ? type_network_name
+    'enable_experimental_unrecoverable_data_corrupting_features' ? string[1..]
+    'filestore_xattr_use_omap'  : boolean = true
+    'fsid'                      : type_uuid # Should be generated with uuidgen
+    'mon_cluster_log_to_syslog' : boolean = true
+    'mon_initial_members'       : string [1..]
+    'mon_osd_min_down_reporters' ? long(0..)
+    'mon_osd_min_down_reports'   ? long(0..)
     # the component sets this to false when generating crushmap itself, and true when crushmap is generated by ceph
     'osd_crush_update_on_start' ? boolean 
-    'mon_cluster_log_to_syslog' : boolean = true
+    'osd_journal_size'          : long(0..) = 10240
     'osd_objectstore'           ? string
+    'osd_pool_default_min_size' : long(0..) = 2
+    'osd_pool_default_pg_num'   : long(0..) = 600
```

Comment:
no idea, these have been set a long time ago :smile:  and since this is not used by ceph when creating pools, I will just make it optional